integrations/linear/linear/client.py

```diff
@@ -0,0 +1,171 @@
+from enum import StrEnum
+from typing import Any, AsyncGenerator, Optional
+
+from httpx import Timeout
+
+from loguru import logger
+import jinja2
+from linear.queries import QUERIES
+
+from port_ocean.context.ocean import ocean
+from port_ocean.utils import http_async_client
+
+
+class LinearObject(StrEnum):
+    TEAMS = "TEAMS"
+    LABELS = "LABELS"
+    ISSUES = "ISSUES"
+
```

Comment:
Isnt it duplicated?

Reply:
No, this enum is used to template the strings used to access the saved GraphQL queries (it is different from the `kind` enum)